ksql-streams/src/main/java/io/confluent/ksql/execution/streams/StreamAggregateBuilder.java

```diff
@@ -187,6 +191,19 @@ private StreamAggregateBuilder() {
             resultSchema
         );
 
+    reduced = reduced.transformValues(
```

Comment:
Humm... well, introduction of structured keys is very likely to be a breaking change anyway. However, I can see your point.  This would fit well as a separate step, but I'm up against it on the clock.  Would you have time to split this? (Wait for the follow up PR before you do).  Otherwise, I'll add it to my list and do my best to get it done before the release.

Reply:
Thinking on this more... I'm not sure adding another step really helps in this situation. 

If we add the extra step, then once we have structured keys and the window bounds columns can just be in the key schema, then we'd no longer need the extra step.  New plans could just omit it, but existing plans would still contain it.  So the step would likely become a no-op, but we'd need to keep it around for compatibility sakes

Contrast this to what would happen if we kept this where it is.  When structured keys is done we can just delete this bit of code from here.  New and old plans will now still have the same steps and will still both work.  No need for some legacy step to be kept around.